business_requirement_deliverable/models/project.py

```diff
@@ -15,3 +15,10 @@ class Project(models.Model):
         Deliverables linked to this project.
         Currency of the Deliverables will be the one from this pricelist.'''
     )
+
+    @api.multi
+    def get_closest_ancestor_pricelist(self):
+        for project in self:
+            pricelist = self.env['product.pricelist'].browse(
+                project.pricelist_id.id)
```

Comment:
@elicoidal Seb told me that the rule had not been confirmed, So just let me keep the function and in other model can use this function.